src/main/java/org/broadinstitute/hellbender/utils/recalibration/RecalibrationReport.java

```diff
@@ -325,12 +325,24 @@ else if (argument.equals("binary_tag_name"))
                 RAC.BINARY_TAG_NAME = (value == null) ? null : (String) value;
 
             else if (argument.equals("sort_by_all_columns"))
-                RAC.SORT_BY_ALL_COLUMNS = Boolean.parseBoolean((String) value);
+                RAC.SORT_BY_ALL_COLUMNS = decodeBoolean(value);
         }
 
         return RAC;
     }
 
+    private byte decodeByte(final Object value) {
+        return value instanceof Byte ? (Byte)value: Byte.parseByte((String)value);
```

Comment:
Can you assume that `value` is a `String` if it's not a `Byte`? Could you ever get a `ClassCastException` here? (applies to the new methods below as well).